### PR TITLE
docs: add lumos upgrade doc

### PR DESCRIPTION
**Pull request by sighWang**

### CODE REVIEW

File: docusaurus/docs/run-lumos-on-web.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 2
+---
+
+# Now lumos can run in a web browser
+
+### TL;NR
+
+lumos在最初的设计比较偏向server端，无法在web端运行。为此在web端运行，我们做了用ckb-indexer替换native indexer，增加大数库BI等一系列升级，使得lumos可以运行在web端.
```
Comment (homura) [inline docusaurus/docs/run-lumos-on-web.md]:
```suggestion
lumos 最初运行时只为支持 NodeJS，并没有考虑运行在 web 环境中。为了在 web 端运行，我们用 ckb-indexer 替换了 native indexer，增加大数库 BI 等一系列升级，使 lumos 可以运行在 web 环境下
```

File: docusaurus/docs/run-lumos-on-web.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    balance = balance.add(cell.cell_output.capacity);
+  }
+
+  return balance;
+}
+```
+
+完整的示例请参考 [ckb-indexer-collector example](https://github.com/nervosnetwork/lumos/blob/develop/examples/ckb-indexer-collector.ts)
+
+### indexer
```
Comment (homura) [inline docusaurus/docs/run-lumos-on-web.md]:
```suggestion
### ckb-indexer
```

File: docusaurus/docs/run-lumos-on-web.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  const collector = indexer.collector({ lock });
+
+  let balance: BI = BI.from(0);
+  for await (const cell of collector.collect()) {
+    balance = balance.add(cell.cell_output.capacity);
+  }
+
+  return balance;
+}
+```
```
Comment (homura) [inline docusaurus/docs/run-lumos-on-web.md]:
```sh
npm install @ckb-lumos/lumos
# yarn add @ckb-lumos/lumos
```

```ts
import { Script, Indexer, BI } from "@ckb-lumos/lumos";
async function main(): Promise<BI> {
  const lock = { code_hash: '0x...', hash_type: 'type', args: '0x...' }
  const indexer = new Inexer('http://localhost:8116', 'http://localhost:8114');
  const collector = indexer.collector({ lock });
  let balance: BI = BI.from(0);
  for await (const cell of collector.collect()) {
    balance = balance.add(cell.cell_output.capacity);
  }
  return balance;
}

main();
```

File: docusaurus/docs/run-lumos-on-web.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 2
+---
+
+# Now lumos can run in a web browser
```
Comment (homura) [inline docusaurus/docs/run-lumos-on-web.md]: maybe we can change it to "在浏览器中运行 Lumos"

**Comment by sighWang** [comment]
close this pr cause those file has already been merge in https://github.com/nervosnetwork/lumos/pull/288

**Pull request closed without merging**